zuul.d/nodeset.yaml

```
@@ -326,3 +326,275 @@
     nodes:
       - name: controller
         label: centos-9-stream-crc-2-39-0-xl
+
+
+#
+# CRC-2.48 (OCP4.18) nodesets
+#
+
+- nodeset:
+    name: centos-9-medium-crc-extracted-2-48-0-3xl
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-medium
+      - name: crc
+        label: crc-cloud-ocp-4-18-1-3xl
+    groups:
+      - name: computes
+        nodes: []
+      - name: ocps
+        nodes:
+          - crc
+
+- nodeset:
+    name: centos-9-crc-2-48-0-xxl
+    nodes:
+      - name: controller
+        label: crc-cloud-ocp-4-18-1-xxl
```

```suggestion
        label: centos-9-stream-crc-2-48-0-xxl
```
This should be nested image right?